src/quo2/components/settings/data_item/style.cljs

```diff
@@ -0,0 +1,68 @@
+(ns quo2.components.settings.data-item.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn container
+  [size card? blur?]
+  {:flex               1
+   :flex-direction     :row
+   :justify-content    :space-between
+   :padding-vertical   (when (= size :default) 8)
+   :padding-horizontal (when (= size :default) 12)
+   :border-radius      16
+   :border-width       (when (and card? (not= size :small)) 1)
+   :border-color       (if
+                         blur?
+                         colors/white-opa-10
+                         (colors/theme-colors colors/neutral-10
```

Comment:
`theme` has to be a 3rd param of `theme-colors` for it to be used correctly. please adjust! 🙏

Reply:
Done